.github/workflows/storage.yml

```diff
@@ -169,7 +166,7 @@ jobs:
       plist_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
       signin_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
       LEGACY: true
-    runs-on: macos-14
+    runs-on: macos-15
```

Comment:
one caveat with this. the default xcode version on macos 15 is Xcode 16.0.

while it's not critical for source based builds, we could either:
- explicitly set the Xcode version for all macos 15 jobs
- only set it when needed (build uses binary built with Xcode 16.2)